starport/pkg/spn/proposal.go

```diff
@@ -216,30 +216,21 @@ func (c *Client) Propose(ctx context.Context, accountName, chainID string, propo
 
 		switch {
 		case proposal.Account != nil:
-			addr, err := types.AccAddressFromBech32(proposal.Account.Address)
-			if err != nil {
-				return err
-			}
-
 			// Create the proposal payload
 			payload := launchtypes.NewProposalAddAccountPayload(
-				addr,
+				proposal.Account.Address,
 				proposal.Account.Coins,
 			)
 
 			msgs = append(msgs, launchtypes.NewMsgProposalAddAccount(
 				chainID,
-				clientCtx.GetFromAddress(),
+				clientCtx.GetFromAddress().String(),
 				payload,
 			))
 
 		case proposal.Validator != nil:
 			// Get the validator address
-			addr, err := types.AccAddressFromBech32(proposal.Validator.ValidatorAddress)
-			if err != nil {
-				return err
-			}
-			validatorAddress := types.ValAddress(addr)
+			validatorAddress := types.ValAddress(proposal.Validator.ValidatorAddress).String()
```

Comment:
We must directly use `proposal.Validator.ValidatorAddress` as the validator address since the validator address format is no longer recognized on `spn`

Reply:
solved by 56781c8.